packages/flutter_tools/test/integration.shard/README.md

```diff
@@ -11,7 +11,7 @@ Use this command to run (from the `flutter_tools` directory):
 ```
 
 These tests are expensive to run and do not give meaningful coverage
-information for the flutter tool (since they are black-box tests that
+information for the Flutter tool (since they are black-box tests that
```

Comment:
the tool's name is lowercase (but if you like you can put it in backticks, as in "the \`flutter\` tool") to make it clearer)

Reply:
Thanks! That confused me - I also added a section on how to download the Dart SDK into the path.